test/jest_integration/saml_multiauth.test.ts

```diff
@@ -235,8 +236,8 @@ describe('start OpenSearch Dashboards server', () => {
   });
 
   it('Login to Dashboards and resume from nextUrl', async () => {
```

Comment:
The objective of the test is to verify whether users can successfully resume their session on the last page they visited. Given this aim, it is more practical to use a testing URL that points to a simpler page, one that can be loaded with greater ease and reliability. The previous URL directed to the sample flights data visualization page, which, with its comprehensive metrics and data, necessitated loading extensive framework components—this complexity introduced flakiness into the test. Thats the reason why, instead of testing out the visualization url, I was switching to the index management page which is relatively easy structured.